transformers/storage/cat/initializer/initializer.go

```diff
@@ -27,6 +28,7 @@ import (
 
 var catAddress = constants.GetContractAddress("MCD_CAT")
 var StorageTransformerInitializer transformer.StorageTransformerInitializer = storage.Transformer{
+	Address:           common.HexToAddress(constants.GetContractAddress("MCD_CAT")),
 	HashedAddress:     types.HexToKeccak256Hash(constants.GetContractAddress("MCD_CAT")),
```

Comment:
Not really a big deal but I'm wondering if we might want to only inject the `Address` and then make `HashedAddress` private/derive it from the `Address`. I think it'd be hard to make this mistake but it'd be a bummer if we accidentally passed in different constants to `HexToAddress` vs `HexToKeccak256Hash`

Reply:
Yep, I really like that idea! I'll make that change in this PR and in makerdao/vulcanizedb#38